src/services/juxt-web/routes/console/communities.ts

```diff
@@ -107,19 +120,22 @@ router.get('/:communityID/:type', async function (req, res) {
 
 	if (!community.permissions) {
 		community.permissions = {
-			open: community.open,
+			open: false,
```

Comment:
The `Community` type doesn't have `open` on it, I think this must have been an older format of model before `permissions` was added. I can add this back in and just do something like `(community as any).open` to maintain the functionality here? Could also check if there are any communities that haven't actually been migrated by this script in the DB and just remove the whole thing if it's not required

Reply:
Iirc yes this is done because some older documents were made before `permissions`, and this was a way to just migrate documents on the fly. Would have to verify with @CaramelKat but I believe that was the use case here.

I do agree that a standard migration would be better however, assuming that any even still need to be migrated.

In cases where a PR requires a database migration we also prefer to list the migration steps/query in the PR message, btw. See https://github.com/PretendoNetwork/friends/pull/15 for reference. I may add this to the contributing doc as well so it's clear.